lib/services/livesync/ios-platform-livesync-service.ts

```diff
@@ -29,10 +29,11 @@ class IOSPlatformLiveSyncService extends PlatformLiveSyncServiceBase {
 					let localToDevicePaths = this.$projectFilesManager.createLocalToDevicePaths(deviceAppData, projectFilesPath, null, this.liveSyncData.excludedProjectDirsAndFiles);
 					let afterSyncAction: () => IFuture<void>;
 
+					this.transferFiles(deviceAppData, localToDevicePaths, this.liveSyncData.projectFilesPath, true).wait();
```

Comment:
how will this code work in case the application is not installed on device? Where will it transfer the files as there's no sandbox on device (`/Library/Application Support` directory for current app)?